backend/apps/packages/services.py

```
@@ -0,0 +1,590 @@
+            json=payload,
+            timeout=FIREWORKS_TIMEOUT_SECONDS,
+        )
+        response.raise_for_status()
+    except requests.RequestException as exc:
+        raise LabelAIError("No se pudo procesar la etiqueta con Fireworks AI.") from exc
+
+    body = response.json()
+    try:
+        content = body["choices"][0]["message"]["content"]
+    except (KeyError, IndexError, TypeError) as exc:
+        raise LabelAIError("La respuesta de Fireworks AI no tiene el formato esperado.") from exc
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)